packages/ui/src/components/pages/SetupWallet/NewWallet/BackupWallet.tsx

```diff
@@ -0,0 +1,100 @@
+import { ChangeEvent, FormEvent, useMemo, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useDispatch, useSelector } from 'react-redux';
+import { useEffectOnce } from 'react-use';
+import { generateMnemonic } from '@polkadot/util-crypto/mnemonic/bip39';
+import { WalletBackup } from '@coong/keyring/types';
+import { LoadingButton } from '@mui/lab';
+import { Button, Checkbox, FormControlLabel, FormGroup } from '@mui/material';
+import BackupSecretRecoveryPhrase from 'components/pages/SetupWallet/NewWallet/BackupSecretRecoveryPhrase';
+import JsonFile from 'components/shared/export/JsonFile';
+import QrCode from 'components/shared/export/QrCode';
+import CryptoJS from 'crypto-js';
+import useSetupWallet from 'hooks/wallet/useSetupWallet';
+import { setupWalletActions } from 'redux/slices/setup-wallet';
+import { RootState } from 'redux/store';
+import { Props, TransferableObject, BackupWalletMethod } from 'types';
+
+interface BackupWalletProps extends Props {
+  method: BackupWalletMethod;
+  resetMethod: () => void;
+}
+
+interface BackupProps extends Props {
+  method: BackupWalletMethod;
+}
+
+function Backup({ method }: BackupProps): JSX.Element {
+  const { password, secretPhrase } = useSelector((state: RootState) => state.setupWallet);
+
+  // Preventing re-create the wallet backup when the component is re-rendered
+  const walletBackup = useMemo(() => {
+    const encryptedMnemonic = CryptoJS.AES.encrypt(secretPhrase!, password!).toString();
+
+    return {
+      encryptedMnemonic,
+    } as WalletBackup;
+  }, [secretPhrase, password]);
+
+  switch (method) {
+    case BackupWalletMethod.SecretRecoveryPhrase:
+      return <BackupSecretRecoveryPhrase secretPhrase={secretPhrase!} />;
+    case BackupWalletMethod.QrCode:
+      return <QrCode value={walletBackup} object={TransferableObject.Wallet} />;
+    case BackupWalletMethod.Json:
```

Comment:
Title: Downloads JSON file
Description: Downloads the below JSON file and upload it to the cloud (iCloud, Google Drive, ...), you can restore your wallet by uploading this JSON file to Coong Wallet later at any time with your wallet password.